docs/programming_guide/execution_api_type.rst

```diff
@@ -1,11 +1,12 @@
 .. _execution_api_type:
 
-##################
-Execution API Type
```

Comment:
Let me update the MONAI config and 3rd party integration port documentation as well